ui/src/config/invenioConfig.js

```diff
@@ -101,6 +101,7 @@ export const invenioConfig = {
   ],
   rest_mimetype_query_arg_name: 'format',
   support_email: 'info@inveniosoftware.org',
+  order: { defaultCurrency: 'CHF' },
```

Comment:
maybe this should be config, not hardcoded

Reply:
it is config, no?

Reply:
@kprzerwa @ntarocco, just to avoid any confusion here, i placed it in `invenioConfig` cause I considered it config, @kprzerwa are you talking about putting it in `config.py` (it doesn't feel appropriate)? @ntarocco I am guessing you state that it is ok to leave it here.